bin/ream/src/cli/lean_node.rs

```diff
@@ -15,4 +18,20 @@ pub struct LeanNodeConfig {
       value_parser = lean_network_parser
   )]
     pub network: Arc<LeanNetworkSpec>,
+
+    #[arg(
+        default_value = "default",
+        long,
+        help = "One or more comma-delimited base64-encoded ENR's of peers to initially connect to. Use 'default' to use the default bootnodes for the network. Use 'none' to disable bootnodes."
+    )]
+    pub bootnodes: Bootnodes,
+
+    #[arg(long, help = "Set P2P socket address", default_value_t = DEFAULT_SOCKET_ADDRESS)]
+    pub socket_address: IpAddr,
+
+    #[arg(long, help = "Set P2P socket port (TCP)", default_value_t = DEFAULT_SOCKET_PORT)]
+    pub socket_port: u16,
+
+    #[arg(long, help = "Discovery 5 listening port (UDP)", default_value_t = DEFAULT_DISCOVERY_PORT)]
+    pub discovery_port: u16,
```

Comment:
```suggestion
```

There are no plans to use discv5 yet, so we shouldn't add a config argument for it